crates/vm/levm/src/memory.rs

```diff
@@ -92,8 +92,8 @@ impl Memory {
         let real_new_memory_size = new_memory_size + self.current_base;
 
         if real_new_memory_size > buffer.len() {
-            // when resizing, resize by allocating entire pages instead of small memory sizes.
-            let new_size = real_new_memory_size.next_multiple_of(4096);
+            // when resizing, avoid really small resizes.
+            let new_size = real_new_memory_size.next_multiple_of(64);
```

Comment:
Which values did we try here? Maybe there's a better choice for this constant. Or we could also try changing it dynamically (start with 32, then 64, then 128, etc.).